### PR TITLE
adds json to cms frame name

### PR DESCRIPTION
**Pull request by lory87**
**This pull request fixes the following issues**:

- to get the url of the current page I had to add a json object to the frame name

https://github.com/divio/djangocms-admin-style/pull/183

### CODE REVIEW

**Comment by FinalAngel** [comment]
LGTM

File: cms/static/cms/js/modules/cms.toolbar.js [hunk 1]
```diff
@@ -573,7 +573,7 @@ var CMS = window.CMS || {};
                         });
                         break;
                     case 'cms_frame':
-                        var frame = window.open(el.attr('href'), 'cms_frame');
+                        var frame = window.open(el.attr('href'), JSON.stringify({ name: 'cms_frame', url: window.location.href}));
```
Comment (vxsx) [inline cms/static/cms/js/modules/cms.toolbar.js]: space after `href`

**Comment by vxsx** [comment]
Build is failing because of unrelated spelling mistakes in 3.2 upgrade guide

**Comment by vxsx** [comment]
https://github.com/divio/django-cms/pull/4588

**Pull request merged**